### ISSUE TITLE
flow-process: ensure imports required in handlers are retrieved from their local node-modules and not from flow-process

### ISSUE THREAD

**Issue by hpl002** [stacc/stacc-flow-community#10]
**Relevant Flow module(s)**
Flow-process
 

**What do you want to accomplish? Please describe.**
Had a very lucky accident that went unnoticed since the inception of the project. 

Handler required package. Local package.json did not have this deps declared.. When node does not find the package in current dir then it traverses backwards until package is found. Flow-process uses this same package and we were therefore able to run without problems.
 - https://nodejs.org/api/modules.html#modules_loading_from_node_modules_folders
 
 
However, these dependencies should be completely independent. Using flow-process as a fallback is bad practice, if we can even call it that..

Would be great if we as _silly developers_ could get a helpful error in the event where we slip up and forget to import required packages locally so that the project does not break when flow-process inevitably updates or removes its deps :)) 

**Describe the suggested solution**
Quick google returned this result -> https://stackoverflow.com/questions/32455431/prevent-require-from-looking-up-modules-in-the-parent-directory

Could try prevent it from looking for packages installed by flow-process? 

**Additional context**
 How this is somewhat clear.
 
 Not urgent and purely the result of bad practice, but could be a nice safety net

**Comment by Anders-eide-stacc** [comment]
Good catch, I am thinking that this will probably be introduced as a breaking change in the next major version of flow-process, but that I will also introduce a new opt-in feature flag for existing v10.X deployments.

**Comment by hpl002** [comment]
Just noticed the exact same issue when upgrading the caseworker portal to 10.x. In this case we did not have axios installed. Should maybe be addressed here as well
